src/main/java/simpaths/experiment/SimPathsMultiRun.java

```diff
@@ -50,7 +50,7 @@ public class SimPathsMultiRun extends MultiRun {
 	private static Map<String, Object> innovationArgs;
 	private static Map<String, Object> collectorArgs;
 	private static Map<String, Object> parameterArgs;
-	public static String configFile = "default.yml";
+	public static String configFile = "test_run.yml";
```

Comment:
It might not matter, but would suggest leaving this as 'default.yml' in the standard files? The github integration tests use 'test_run.yml' as given explicitly as an argument. This would default all further users to the test config file if no argument were given. I think a major difference is that the test one doesn't persist Persons/Households/BenefitUnits csv files (as not required by tests), so new users may be puzzled where these go?